tests/test_zfin.py

```diff
@@ -36,5 +36,18 @@ def test_allZPAvailable(self):
 
         return
 
+    def test_load_zp_mappings(self):
+        if self.source is not None:
+            try:
+                zp_map = self.source._load_zp_mappings("./tests/resources/zfin/zp-mapping-test.txt")
+                self.assertIsInstance(zp_map, dict, "_load_zp_mappings() didn't return dict!")
+                self.assertTrue(len(zp_map) == 1, "_load_zp_mappings() didn't return exactly one thing!")
+                self.assertDictEqual(zp_map, {'MONARCH:b311b709bf1cf852d770': {'zp_id': 'ZP:0002959', 'subterm1_id': 'ZFA:0009114', 'post_composed_relationship_id_1': 'BFO:0000050', 'superterm1_id': 'ZFA:0001056', 'quality_id': 'PATO:0001453', 'subterm2_id': 'GO:0005927', 'post_composed_relationship_id_2': 'BFO:0000050', 'superterm2_id': 'ZFA:0001056'}}, "_load_zp_mappings() didn't return what I expected!")
+            except Exception as e:
```

Comment:
Going to ask that __all__ new variables have a minimum of three characters  
unless contained on one line, such as within a list comprehension. 
(replacing existing single char vars as you go is also highly appreciated)

Reply:
Thanks, done, and I agree - will replace these going forward if/when I come across them